--- Alien-wxWidgets-0.43/Build.PL.orig	2009-05-10 09:48:23.000000000 +0200
+++ Alien-wxWidgets-0.43/Build.PL	2009-06-15 11:21:53.917665314 +0200
@@ -91,9 +91,7 @@
         }
     }
 }
-chomp $build_prompt;
-my $build_wx = _askyn( $build, 'build_wx',
-                       $build_prompt, $build_wx_dflt );
+my $build_wx;
 $build->notes( 'build_wx' => $build_wx );
 $build->notes( 'mk_portable' => $build->args('mk_portable') );
 if( $build_wx ) {
